### DISCUSSION TITLE
security

### DISCUSSION THREAD

**Discussion by ivarbratberg** [icsharpcode/ILSpy#2171] [General] [answered]
Hi,
have you seen this ?
https://www.virustotal.com/gui/file/7b0b8bf198821c47c83eb3428f4af81dae9f8b31be6e0e885719db937a37cd9c/detection

I am not allowed to use dnsoy because of this.

**Comment by christophwille** [comment, marked as answer]
What's a dnSpy report to do with us?

**Reply by ivarbratberg** [reply]
Ah sorry, I have confused ilSpy and dnspy
